source/MaterialXRender/CgltfLoader.cpp

```diff
@@ -402,7 +456,10 @@ bool CgltfLoader::load(const FilePath& filePath, MeshList& meshList, bool texcoo
         }
     }
 
-    cgltf_free(data);
+    if (data)
```

Comment:
This is minor, but static analysis packages will flag this change with a warning, since we dereference `data` in the logic above, without checking for nullptr.  I'd recommend leaving this call to `cgltf_free` unguarded, since it doesn't appear possible to reach this point with a null pointer.

Reply:
Yes. Your correct. Code change has been reverted.